packages/ra-ui-materialui/src/layout/MenuItemLink.tsx

```diff
@@ -95,12 +94,18 @@ export const MenuItemLink = forwardRef((props: MenuItemLinkProps, ref) => {
         [dispatch, isSmall, onClick]
     );
 
+    const match = useMatch(
+        typeof props.to === 'string' ? props.to : props.to.pathname
+    );
+
     const renderMenuItem = () => {
         return (
             <StyledMenuItem
-                className={classnames(MenuItemLinkClasses.root, className)}
-                activeClassName={MenuItemLinkClasses.active}
```

Comment:
The detection of active links is no longer built-in react-router?

Reply:
Not compatible with mui styled components